yarn.lock

```diff
@@ -2,115 +2,208 @@
 # yarn lockfile v1
 
 
+"@ampproject/remapping@^2.1.0":
```

Comment:
Can you uncommit this please. You have mutated the yarn lock. Simply running yarn on your local shouldn't change this at all unless you deleted and recommited it locally.